notebooks/05-toc-line-analysis.ipynb

```diff
@@ -0,0 +1,1709 @@
+      "text/plain": [
+       "<Figure size 1152x1152 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "table = np.zeros(shape=(5, 4))\n",
+    "table[4,3] = tier_4[(tier_4.mode_a==\"metro\") & (tier_4.mode_b.isin([\"metro\", \"bus\"]))].num_TOC.sum()\n",
+    "table[4,2] = tier_4[tier_4.mode_a==\"metro\"].num_TOC.sum()\n",
```

Comment:
I think this might double count some of the metro rail stop entitlements. That is, I think tier 4 metro stops would also be included in the tier 3 metro stop criterion here.

Reply:
I checked with: `tier_3[tier_3.AIN.isin(tier_4.AIN)]` and `tier_4[tier_4.AIN.isin(tier_3.AIN)]` and both came up with empty dataframes. The previous line, where `tier_3 = df[df.TOC_Tier==3` takes care of that, because parcels will only belong in 1 tier. But, in my next commit, I added something to check whether parcels are linked to more than 1 tier, and to always keep the higher tier.

Reply:
I still think there is a problem here. Shouldn't `table[4,2]` be looking at `tier_3`, not `tier_4`?

Reply:
Ahhh, gotcha, fixing it in my new branch.